### ISSUE TITLE
Travis offline since November 6?

### ISSUE THREAD

**Issue by dondi** [dondi/GRNsight#820]
Travis CI seems to have stopped checking builds after November 6.

**Comment by kdahlquist** [comment]
@dondi, has this been resolved?

**Comment by dondi** [comment]
Not yet, and I was anticipating that this would be one of the next ones after #803. I’ll dig around now.

**Comment by dondi** [comment]
I looked at this tonight and it turns out that Travis has still been checking builds. What has been lost is the ability for Travis to report the results of the build back to GitHub so that the ❌ and ✅ symbols (with corresponding links) appear next to the commits.

Upon investigating this further, it looks like GitHub changed the mechanism by which external sites can report back commit results (from “GitHub Services” to “GitHub Apps”). I tried following some documentation for migrating Travis to the newer “GitHub Apps” approach, but have not succeeded yet. I’ll investigate further and report back.

If one connects directly to travis.org, one does see build results. So for now, that’s the way to do this.

**Comment by dondi** [comment]
As it turns out, virtually the very next thing I tried after reporting the previous comment did restore the reporting-back functionality. I tried a commit on my branch that now shows the expected icons.

I’ll set this issue to  “review requested” through a few more commits/pushes to make sure that the integration is fully restored. If we see Travis feedback consistently through this next week of work, then the issue can be closed.
